src/js/view/view.js

```diff
@@ -340,6 +340,45 @@ function View(_api, _model) {
         playerViewModel.change('playlistItem', onPlaylistItem);
         // Triggering 'resize' resulting in player 'ready'
         _lastWidth = _lastHeight = null;
+
+        // Setup floating scroll handler
+        const topOffset = 62;
+        if (_floatingConfig && _api.getEnvironment().OS.mobile) {
+            let canFire = true;
+            let debounceTO;
+
+            const checkFloat = () => {
+                const floating = _model.get('isFloating');
+                const hasCrossedThreshold = playerBounds.top < topOffset ?
+                    playerBounds.top <= window.scrollY :
+                    playerBounds.top <= window.scrollY + topOffset;
+
+                if (!floating && hasCrossedThreshold) {
+                    _updateFloating(1, true);
```

Comment:
not in love with this but felt like over-refactoring to pull all but one line into a sep func